lib/components/RoundButton/RoundButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { useId } from 'react';
+import classNames from 'classnames';
+
+export interface RoundButtonProps {
+  /** Text shown on the button */
+  label: string;
+  /** Callback fired on tap/click of the button */
+  onClick: () => void;
+  /** Button disabled for any actions */
+  disabled?: boolean;
+  /** Selected */
+  selected?: boolean;
+  /** CSS classname */
+  className?: string;
+}
+
+export const RoundButton = ({ label, onClick, disabled = false, selected = false, className }: RoundButtonProps) => {
+  const labelId = useId();
+  return (
+    <div
+      className={classNames('jod-roundbutton', className, 'flex flex-col items-center justify-center px-1', {
```

Comment:
Do we need `jod-roundbutton` css class? If we do, `jod-roundbutton` should be `jod-round-button`, like the name of the component.

Reply:
Helps to find the component from the codebase especially if not familiar with it or so.
We can rename it for sure; personally it felt better visually and easier to read without the dash.

Reply:
I understand the point, but I think it's useless CSS class without any style definitions. I would use [React Developer Tools](https://react.dev/learn/react-developer-tools) inspector to find component from the codebase.

Reply:
I'll remove it.